cdap-watchdog/src/main/java/io/cdap/cdap/logging/pipeline/LogProcessorPipelineContext.java

```diff
@@ -193,4 +193,9 @@ public void increment(String metricName, long value) {
   public void gauge(String metricName, long value) {
     metricsContext.gauge(String.format("%s.%s", metricsPrefix, metricName), value);
   }
+
+  @Override
+  public void event(String metricName, long value) {
+    // TODO
```

Comment:
Shouldn't this just delegate to `metricsContext`?

Reply:
Thanks. I didn't notice it was that simple. Was planning to add later.